typescript/src/constants/roles.ts

```diff
@@ -6,14 +6,22 @@ export enum Role {
   SUPPLIER = 4,
   CLIENT = 5,
   MANAGER = 6,
-  ACCOUNTANT = 7
+  ACCOUNTANT = 7,
 }
 
-export const MANAGEMENT_TEAM = [
+export const DOMAIN_MANAGERS = [
   Role.ADMIN,
+  Role.MANAGER,
+];
+
+export const MANAGEMENT_TEAM = [
+  ...DOMAIN_MANAGERS,
   Role.SALES,
   Role.DESIGNER,
-  Role.SUPPLIER,
-  Role.MANAGER,
-  Role.ACCOUNTANT
+  Role.ACCOUNTANT,
+];
+
+export const BUSINESS_ACCOUNTS = [
+    ...MANAGEMENT_TEAM,
+    Role.SUPPLIER
```

Comment:
you've added trailing `,` to every exiting array, but not to the new one you created?

obviously not a major issue but i will prefer if every pr is smaller and more consistent.

Reply:
added the final comma.